pkgs/games/anura/default.nix

```diff
@@ -0,0 +1,30 @@
+{ SDL2 , SDL2_image, SDL2_ttf, SDL2_mixer
+, stdenv, pkgconfig, icu, boost, glew, cairo
+, fetchgit, which
+}:
+stdenv.mkDerivation {
+  name = "anura";
+  version = "20171110";
+
+  nativeBuildInputs = [  pkgconfig icu.dev which ];
+  buildInputs = [ boost SDL2.dev SDL2_image SDL2_mixer SDL2_ttf glew cairo ];
+
+  installPhase = ''
+      mkdir -p $out/bin
+      install -p anura $out/bin/anura
+  '';
+
+  src = fetchgit {
```

Comment:
`fetchFromGitHub` now also has `fetchSubmodules`, so it's better to use it instead:

```
src = fetchFromGitHub {
  owner = "anura-engine";
  repo = "anura";
  rev = "fee247c573005777de9f6eb67debc83988818898";
  sha256 = "03j6ld0g8kh5x6d2qna1rrp8jzs587a5zw2qz71w5ryqn8w4jdil";
  fetchSubmodules = true;
};
```